resources/leiningen/new/kraken_works/PROJECT@.service.template

```diff
@@ -26,11 +24,11 @@ ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 
 ExecStart=/bin/bash -c 'docker run --name ${CONTAINER} \
   --link rabbitmq:rabbitmq \
```

Comment:
I noticed that while working on the http-api template. I'll get that fixed here.